cmd/influxd/run/server_helpers_test.go

```diff
@@ -358,7 +382,7 @@ func NewTest(db, rp string) Test {
 func (t Test) duplicate() Test {
 	test := Test{
 		initialized: t.initialized,
-		write:       t.write,
+		writes:      t.writes.duplicate(),
 		params:      t.params,
```

Comment:
This will be a bug a swell, we need to `duplicate` the params otherwise every test is referencing the same pointer of params.  This bug wasn't introduced by your changes, but became apparent because of your changes.